kubetest/e2e.go

```diff
@@ -35,8 +35,13 @@ func run(deploy deployer, o options) error {
 		os.Setenv("KUBECTL", "./cluster/kubectl.sh")
 	}
 	os.Setenv("KUBE_CONFIG_FILE", "config-test.sh")
+
 	// force having batch/v2alpha1 always on for e2e tests
-	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
+	runtimeConfig := "batch/v2alpha1=true"
+	if o.runtimeConfig != "" {
+		runtimeConfig = runtimeConfig + "," + o.runtimeConfig
+	}
+	os.Setenv("KUBE_RUNTIME_CONFIG", runtimeConfig)
```

Comment:
I would prefer we just set the var to the flag's value if it's non-empty and then default the flag value to batch/v2alpha1=true